src/legacy/server/kbn_server.d.ts

```diff
@@ -34,6 +34,10 @@ export interface KibanaConfig {
   has(key: string): boolean;
 }
 
+export interface UiApp {
```

Comment:
__note:__ added just what was needed for me to not add too much of `@ts-ignore`.